.github/workflows/unit_tests.yml

```diff
@@ -34,9 +34,9 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies using pip
+    - name: Install Python dependencies using pip
       run: |
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade pip wheel
```

Comment:
why do you need wheel?

Reply:
The Matplotlib dependencies wouldn't install properly otherwise, if I remember correctly. Can try to remove wheel again and see if the issue persists

Reply:
Oh, no, so the thing was without wheel obviously we cannot install wheels, which makes things difficult with matplotlib.